### PR TITLE
add apply-cache

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qw4990** [comment]
PTAL @yikeke @XuHuaiyu

**Review by XuHuaiyu — COMMENTED** [review]
LGTM

**Review by AilinKid — COMMENTED** [review]
/lgtm

File: system-variables.md [hunk 1]
```diff
@@ -792,6 +792,13 @@ v5.0.0-rc 后，用户仍可以单独修改以上系统变量（会有废弃警
 - 这个变量用来设置一条查询语句的内存使用阈值。
 - 如果一条查询语句执行过程中使用的内存空间超过该阈值，会触发 TiDB 启动配置文件中 OOMAction 项所指定的行为。该变量的初始值由配置项 [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query) 配置。
 
+### `tidb_mem_quota_apply_cache`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：32 MB
+- 这个变量用来设置 Apply 算子中局部 Cache 的内存使用阈值。
+- Apply 算子中局部 Cache 用来加速 Apply 算子的计算, 该变量可以设置其内存使用阈值。设置其为 `0` 可以关闭 Apply Cache 功能。
```
Comment (TomShawn) [inline system-variables.md:800]:
```suggestion
- `Apply` 算子中局部 Cache 用来加速 `Apply` 算子的计算，该变量可以设置 `Apply` Cache 的内存使用阈值。设置变量值为 `0` 可以关闭 `Apply` Cache 功能。
```

File: system-variables.md [hunk 2]
```diff
@@ -792,6 +792,13 @@ v5.0.0-rc 后，用户仍可以单独修改以上系统变量（会有废弃警
 - 这个变量用来设置一条查询语句的内存使用阈值。
 - 如果一条查询语句执行过程中使用的内存空间超过该阈值，会触发 TiDB 启动配置文件中 OOMAction 项所指定的行为。该变量的初始值由配置项 [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query) 配置。
 
+### `tidb_mem_quota_apply_cache`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：32 MB
+- 这个变量用来设置 Apply 算子中局部 Cache 的内存使用阈值。
```
Comment (TomShawn) [inline system-variables.md:799]:
```suggestion
- 该变量用来设置 `Apply` 算子中局部 Cache 的内存使用阈值。
```

**Pull request merged**